src/application/cli.py

```
@@ -0,0 +1,205 @@
+    if args.config.exists():
+        config = ApplicationConfig.from_file(args.config)
+    else:
+        config = ApplicationConfig.from_env()
+    
+    setup_logging(
+        level=config.logging.level,
+        format_type=config.logging.format,
+        log_file=config.logging.file_path
+    )
+    
+    container = get_container()
```

Ruff (0.12.2)

Local variable `container` is assigned to but never used

Remove assignment to unused variable `container`

(F841)